### PR TITLE
feat: drizzle integration and schemas

### PR DESCRIPTION
**Pull request by CedricAngeloSina-on**
-Drizzle ORM
-Drizzle Studio 

-define db credentials (env vars) in drizzle.config.ts 
-import the 0000_skinny_veda.sql in /migrations to a local postgresdb
-to open drizzle studio
--"pnpm drizzle-kit studio" 

-Defined schemas
--admins
--items
--employees
--managers
--users
--audit_histories (not finished)

### CODE REVIEW

**Comment by CedricAngeloSina-on** [comment]
decided to cancel, needs to code review

**Review by davidrayc — APPROVED** [review]

**Review by davidrayc — CHANGES_REQUESTED** [review]
Change target branch to `develop` instead of to `main`

**Comment by CedricAngeloSina-on** [comment]
changed branch target to `develop`

**Review by davidrayc — APPROVED** [review]

File: db/drizzle/migrations/0000_skinny_veda.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
+CREATE TABLE IF NOT EXISTS "admins" (
```
Comment (evad-e) [inline db/drizzle/migrations/0000_skinny_veda.sql:1]: Let's follow SQL naming convention:
1.) Table names should be singular
2.) Column names should be singular
3.) Column names should be snake case

This also applies to schema file and variable names.

Comment (evad-e) [reply db/drizzle/migrations/0000_skinny_veda.sql:1]: >0000_skinny_veda.sql
>0001_clean_pete_wisdom.sql
>0002_lethal_tombstone.sql

Not really familiar with migrations in NextJS, but ganito ba yung naming nila? Just curious hahaha

**Review by evad-e — CHANGES_REQUESTED** [review]

File: db/drizzle/migrations/0000_skinny_veda.sql (hunk 1 above)
Comment (CedricAngeloSina-on) [reply db/drizzle/migrations/0000_skinny_veda.sql:1]: auto generated file names lang yan ng drizzle magmigrate ng db

Comment (CedricAngeloSina-on) [reply db/drizzle/migrations/0000_skinny_veda.sql:1]: you can delete (sql files + meta folder) and migrate again to create a fresh .sql file
command: pnpm drizzl-kit generate:pg

Comment (evad-e) [reply db/drizzle/migrations/0000_skinny_veda.sql:1]: > auto generated file names lang yan ng drizzle magmigrate ng db

Ohhh ang cool namn hahaha

Comment (evad-e) [reply db/drizzle/migrations/0000_skinny_veda.sql:1]: > you can delete (sql files + meta folder) and migrate again to create a fresh .sql file command: pnpm drizzl-kit generate:pg

Sige, please push your changes para ma approve ko thanks!

**Review by evad-e — APPROVED** [review]

**Pull request merged**